recipe/win_forwarder.c

```diff
@@ -0,0 +1,103 @@
+// adapted from example in MSFT docs, see
+// https://learn.microsoft.com/en-us/windows/win32/procthread/creating-processes
```

Comment:
What's the license on this?

Reply:
MIT, see [here](https://github.com/MicrosoftDocs/win32/blob/6ee422bccf4e29536446ccc72fec9ccf5fa4755a/ThirdPartyNotices#L1-L5), which covers the code in doc [source](https://github.com/MicrosoftDocs/win32/blob/docs/desktop-src/ProcThread/creating-processes.md)

Reply:
Any other concerns besides the license?

Reply:
Gentle ping on this @isuruf

Reply:
Another ping here... I'm aware this is a trade-off between consistency across platforms and extra complexity (though we hopefully shouldn't have to touch it going forward; might also consider breaking this out into something reusable wherever we have versioned binaries). I don't have a clear answer what is better, so I'd appreciate your input @isuruf.

Reply:
Can we just use symlink-exe-substitute-feedstock ?

Reply:
It looks like [that's](https://github.com/conda-forge/symlink-exe-substitute-feedstock?tab=readme-ov-file) built to symlink an exe into another folder, not to a differently-named file in the same folder (it also doesn't use the relative path logic). That said, the approach could probably be extended to name the target symlink, and it's cleaner because it doesn't duplicate the 12kb per link (at the cost of introducing an extra run-dependency).